src/components/Nav/Nav.tsx

```diff
@@ -99,6 +105,7 @@ class Nav extends Component<Props, State> {
         <div className={style.langSelector}>
           <div className={cn(lang === 'ru' && style.active)} onClick={this.langChange('ru')}>Ru</div>
           <div className={cn(lang === 'en' && style.active)} onClick={this.langChange('en')}>En</div>
+          <div className={cn(lang === 'zh' && style.active)} onClick={this.langChange('zh')}>Zh</div>
```

Comment:
Why `Zh` and not `Cn`?

Reply:
Languague codes https://ru.wikipedia.org/wiki/%D0%9A%D0%BE%D0%B4%D1%8B_%D1%8F%D0%B7%D1%8B%D0%BA%D0%BE%D0%B2 here Chinese is zh. Also, I found examples which work on zh model and don't work on the multi_lang. Should I add this examples?